src/components/Login/Login.js

```diff
@@ -17,35 +25,41 @@ function Signin({ isOpen, onClose, onSubmit, onSignupClick }) {
                     aria-label="close"
                     className="popup__close-button-signin"
                 />
-                <form name="signin" className="popup__form" onSubmit={onSubmit}>
+                <form name="signin" className="popup__form" onSubmit={handleLogin}>
```

Comment:
`PopupWithForm` should contain tag `form`, **close icon** , **submit button** not to duplicate them in every popup with a form

Reply:
Done